readthedocs/templates/core/build_list_detailed.html

```diff
@@ -4,7 +4,7 @@
           {% for build in build_qs %}
           <li class="module-item col-span">
             <div id="build-{{ build.id }}">
-                <a href="{{ build.get_absolute_url }}"><span id="build-state">{% if build.state != 'finished' %}{{ build.get_state_display }} {% else %} {% if build.success %}{% trans "Passed" %}{% else %}{% trans "Failed" %}{% endif %}{% endif %}</span>
+                <a href="{{ build.get_absolute_url }}">{% if build.is_stale %}<span class="icon-warning" title="{% trans 'This build is still waiting to be built' %}"></span>{% endif %}<span id="build-state">{% if build.state != 'finished' %}{{ build.get_state_display }} {% else %} {% if build.success %}{% trans "Passed" %}{% else %}{% trans "Failed" %}{% endif %}{% endif %}</span>
```

Comment:
@humitos  What Do you think about changing `{% if build.state != 'finished' %}` to `{% if build.finished %}` as already there is a property method on Build model named  `Build.finished`? as @agjohnson mentioned to avoid logic on templates.